src/pages/library/datasetBuilder/DatasetBuilder.ts

```diff
@@ -409,10 +409,35 @@ export const DatasetBuilderContents = ({ onStateChange }: { onStateChange: OnSta
       }),
       h(ConceptSetSelector, {
         selectedConceptSets,
-        onChange: (conceptSets) => setSelectedConceptSets(conceptSets),
+        onChange: async (conceptSets) => {
+          setSelectedConceptSets(conceptSets);
+          const uniqueFeatureValueGroups = _.flow(
+            _.flatMap((headerAndValues: HeaderAndValues<ConceptSet>) => headerAndValues.values),
+            _.map((conceptSet: ConceptSet) => conceptSet.featureValueGroupName),
```

Comment:
Could these use `get`?
```suggestion
            _.flatMap(_.get('values')),
            _.map(_.get('featureValueGroupName')),
```

Reply:
They could, but I think I lean towards using the field accessor. I think that would change if the variable name changes via refactor, but `_.get` won't.

Reply:
Makes sense. I like `get` since it means I don't have to write (and later read) a lambda in the source. But it's probably not worth losing typescript enforcement and refactoring support.